test/commands/iteration/analysis/process_test.rb

```diff
@@ -60,4 +60,14 @@ class Iteration::Analysis::ProcessTest < ActiveSupport::TestCase
 
     assert iteration.reload.analysis_exceptioned?
   end
+
+  test "broadcast" do
+    iteration = create :iteration
+    data = { 'status' => :approve, 'comments' => [] }
+
+    IterationChannel.expects(:broadcast!).with(iteration)
+    IterationsChannel.expects(:broadcast!).with(iteration.solution)
```

Comment:
Normally I would have separate tests for the two broadcasts, but I've found that the Ruby error message is so clear (exact expectation that failed) that I think combining them is okay.

Reply:
Yeah - I agree entirely.